tests/integration/mock_data/all.workflow.database.tsv

```diff
@@ -0,0 +1,6 @@
+DATE	CROMWELL_SERVER	RUN_ID	WDL_NAME	STATUS	ALIAS
```

Comment:
There is another copy of this file in tests/unit/mock_data/ , I made a copy here to be used by integrations tests only in the case the files need to diverge in the future. Does that sound reasonable or should they be consolidated?